.changeset/smooth-flies-own.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/modal': patch
+'@twilio-paste/core': patch
+---
+
+Created style file for stylings of the Modal
```

Comment:
It might help to add _why_ you pulled the Modal style out into its own file, for someone reading the Modal changelog who doesn't know about Alert Dialog

Reply:
Counter point, as this is a change log for consumers of the package, they aren't concerned with what you did, but what they can now do with it.